test/v1/connectors/BaseDownstreamConnectorSpec.scala

```diff
@@ -211,7 +211,7 @@ class BaseDownstreamConnectorSpec extends ConnectorSpec {
             .parameterGet(absoluteUrl, params, config, requiredHeaders, excludedHeaders)
             .returns(Future.successful(outcome))
 
-          await(connector.getWithQueryParams(IfsUri[Result](url), params)) shouldBe outcome
```

Comment:
I think we need the TysIfs tests here as well

Reply:
> I think we need the TysIfs tests here in the BaseDownstreamConnectorSpec

BaseDownstreamConnectorSpec doesn't really need them added, there isn't much value (if any) adding extra tests there

Good catch re TysIfsTest in ConnectorSpec though, that was missing a chunk of TYS code